src/utils.php

```diff
@@ -645,3 +649,133 @@ function ensure_dir( $dir ) {
 
 	return realpath( $dir );
 }
+
+/**
+ * Create a file named ".lastruntime" in the root directory and write the current Unix time to it.
+ *
+ * @return bool True if the file was successfully written, false otherwise.
+ */
+function write_last_runtime_to_file(): bool {
+	$filename = '.lastruntime';
+	// Get the current Unix time.
+	$current_unix_time = time();
+
+	// Define the path to the .lastruntime file in the root directory.
+	$file_path = root( "/{$filename}" );
+
+	// Write the current Unix time to the .lastruntime file.
+	$result = file_put_contents( $file_path, $current_unix_time );
```

Comment:
Set the `LOCK_EX` flag here to avoid having two processes trying to write to the same file.
Better safe than sorry when it comes to this type of file.